camel-route-support/src/main/java/io/kaoto/backend/model/deployment/camelroute/Integration.java

```diff
@@ -32,15 +33,13 @@
  *
  */
 
-
-@JsonInclude(JsonInclude.Include.NON_NULL)
+@Version(value = "v1" , storage = true , served = true)
 @Group("camel.apache.org")
-@Version("v1")
-@Kind("Integration")
-@JsonIgnoreProperties(ignoreUnknown = true)
-public final class Integration
-        extends CustomResource<IntegrationSpec, KameletBindingStatus>
-        implements io.fabric8.kubernetes.api.model.Namespaced, Serializable {
+@Singular("integration")
+@Plural("integrations")
+@Generated("io.fabric8.java.generator.CRGeneratorRunner")
+public final class Integration extends CustomResource<IntegrationSpec, IntegrationStatus> implements Namespaced {
```

Comment:
Because of the "flows" property, I had to extend `IntegrationSpec` class too. Which means I can't directly extend the `Integration` class from Camel K CRD.